docs/lago-tutorial/README.md

```diff
@@ -0,0 +1,206 @@
+# lago-tutorial
+This repository contains the default files for creating Lago environment. 
+
+In this tutorial you will learn how to set up a custom virtual environment with Lago.
+The virtual environment will be used as an infrastructure for your application, so you can run tests
+in order to check the functionality of your application, without the overhead of setting up an infrastructure.
+
+This tutorial will be “hands on”, and will take you through the process of creating, running and deploying the environment.
+
+Lago environments are flexible and can be used for all different kinds of tasks. For keeping things simple I will focus on one scenario, although you will get the ability to create your own custom environment.
+We will use only the basic mandatory commands of Lago, for further reading please advise: 
+[Lago Hompage](http://lago.readthedocs.org/en/latest/index.html)
+
+#### Prerequisites:
+
+Install Lago - follow [this](http://lago.readthedocs.org/en/latest/README.html) tutorial for more information
+
+#### The scenario:
+
+Create a Lago environment which will consist of three virtual machines that will host Jenkins infrastructure.
+
+#### The VMs:
+
+*  "vm0-server" - Jenkins server
+*  "vm1-slave" - Jenkins slave
+*  "vm2-slave" - Jenkins slave
+
+#### The network:
+
+All the VMs will connect to the same virtual network.
+Lago allows for a creation of 10 different lans ranging from: 192.168.200.x to 192.168.209.x
+The subnet will be automatically assigend to the virtual network bridge.
+
+#### Creating the working directory:
+
+In order to create a working directory, clone the following repo:
+
+https://github.com/gbenhaim/lago-tutorial
+
+#### What is in the repository?
```

Comment:
I think this should be kept here, as it's common through all the scenarios (they have the init file (init.json, init.yml, LagoInitFile)...)